compiler/src/iree/compiler/Dialect/Flow/Transforms/FusionOfTensorOps.cpp

```diff
@@ -344,123 +344,6 @@ void FusionOfTensorOpsPass::runOnOperation() {
   Operation *funcOp = getOperation();
   MLIRContext *context = funcOp->getContext();
 
-  {
-    RewritePatternSet fusionPatterns(&getContext());
-    // Only fuse operations where all uses of the producer are generic
-    // operations. If an operation is used in a named op, it will be computed
-    // anyway, so the consumers can just use that value.
-    linalg::ControlFusionFn fuseElementwiseOpsControlFn =
-        [&](OpOperand *fusedOperand) {
-          Operation *producer = fusedOperand->get().getDefiningOp();
-          Operation *consumer = fusedOperand->getOwner();
-
-          if (!isNonNullAndOutsideDispatch({producer, consumer})) {
-            return false;
-          }
-
-          // Limit the number of operands. We have hard limit (32) of bindings
-          // passing down to HAL. Set the number to be as same as the limit --
-          // IREE_HAL_MODULE_MAX_DESCRIPTOR_BINDING_COUNT.
-          constexpr int64_t kIreeMaxOperandCount = 32;
-          DenseSet<Value> operands;
-          operands.insert(producer->operand_begin(), producer->operand_end());
-          operands.insert(consumer->operand_begin(),
-                          std::next(consumer->operand_begin(),
-                                    fusedOperand->getOperandNumber()));
-          operands.insert(std::next(consumer->operand_begin(),
-                                    fusedOperand->getOperandNumber() + 1),
-                          consumer->operand_end());
-          if (operands.size() >= kIreeMaxOperandCount)
-            return false;
-
-          return areFusableOps(context, fusedOperand, fuseMultiReduction);
```

Comment:
I do not see a use for this function anymore. It looks like it got moved and renamed to `FusionUtils`, but is still in this file?

Reply:
Ah, looks like I forgot to delete. Good catch!